driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraph.java

```diff
@@ -53,7 +53,7 @@
 import com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage;
 
 @Graph.OptIn(Graph.OptIn.SUITE_STRUCTURE_STANDARD)
-public final class OrientGraph implements Graph {
```

Comment:
I was thinking about removing the final as well, but then I realized the other graph implementation are final...

Any particular reason to do so?